src/nbl/CMakeLists.txt

```diff
@@ -363,6 +363,16 @@ macro(nbl_target_link_gli _trgt)
 	)
 	target_include_directories(${_trgt} PUBLIC ${THIRD_PARTY_SOURCE_DIR}/gli)
 endmacro()
+macro(nbl_target_link_las _trgt)
+	add_dependencies(${_trgt} liblas)
+	target_link_libraries(${_trgt} INTERFACE
+		${THIRD_PARTY_SOURCE_DIR}/las/3rdparty/liblas/liblas/build/bin/$(Configuration)/liblas.lib
+		${THIRD_PARTY_SOURCE_DIR}/las/3rdparty/liblas/liblas/build/bin/$(Configuration)/liblas_c.lib
+	)
+	target_include_directories(${_trgt} PUBLIC 
+		${THIRD_PARTY_SOURCE_DIR}/las/3rdparty/liblas/liblas/include
+	)
```

Comment:
you shouldn't be using paths if you used `add_subdirectory` there should be stuff like named targets i.e. `las::something` that you can use